.gitignore

```
@@ -2,11 +2,12 @@
 build
 debug
 __pycache__
-catalyst.egg-info
+pennylane_catalyst.egg-info
 
-# Testing directories
+# Testing files and directories
 .lit
 .pytest_cache
+_benchmark
```

Any other directories that should go here?
```suggestion
_benchmark
benchmark/datasets
```

---

Sure, will fix. Initially I modified the toplevel `.gitignore` but later I made a special  `benchmark/.gitignore`. I think it would be better to move everything BM-specific there and not touch the global one at all..

---

Oh I see, yes that works too 👍

---

@dime10 I just realized that applying `benchmark/_*` would effectively ignore `benchmark/__init__.py`. Until there is a way to add exception to an exception, I would prefer to just list the working folders explicitly, or maybe invent other prefix pattern rather than `_` to avoid conflicts.